components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodConnectionProvider.kt

```diff
@@ -39,7 +39,6 @@ import io.gitpod.gitpodprotocol.api.entities.WorkspaceInstance
 import io.gitpod.jetbrains.icons.GitpodIcons
 import kotlinx.coroutines.*
 import kotlinx.coroutines.future.await
-import org.bouncycastle.pqc.math.linearalgebra.ByteUtils
```

Comment:
ℹ️  `org.bouncycastle.pqc.math.linearalgebra.ByteUtils` doesn't exist anymore, so we brought `fun toHexString` from [https://github.com/bcgit/bc-java/.../ByteUtils.java#L236-L258](https://github.com/bcgit/bc-java/blob/bc3b92f1f0e78b82e2584c5fb4b226a13e7f8b3b/core/src/main/java/org/bouncycastle/pqc/math/linearalgebra/ByteUtils.java#L236-L258)